src/components/taskDetails/index.tsx

```diff
@@ -0,0 +1,205 @@
+import React, { ChangeEvent, FC } from 'react';
+import NavBar from '@/components/navBar/index';
+import TaskContainer from './TaskContainer';
+import Details from './Details';
+import classNames from './task-details.module.scss';
+import { toast, ToastTypes } from '@/helperFunctions/toast';
+import task from '@/interfaces/task.type';
+import { useState, useEffect, useContext } from 'react';
+import useFetch from '@/hooks/useFetch';
+import { isUserAuthorizedContext } from '@/context/isUserAuthorized';
+import updateTaskDetails from '@/helperFunctions/updateTaskDetails';
+
+type Props = {
+  url: string;
+  taskID: string;
+};
+
+const TaskDetails: FC<Props> = ({ url, taskID }) => {
+  const isAuthorized = useContext(isUserAuthorizedContext);
+  const [isEditing, setIsEditing] = useState<Boolean>(false);
+  const [initialData, setInitialData] = useState<task>();
+  const [taskDetails, setTaskDetails] = useState<any>();
+  const [editedDetails, setEditedDetials] = useState({});
+  const { response, error, isLoading } = useFetch(url);
+  const { SUCCESS, ERROR } = ToastTypes;
+
+  useEffect(() => {
+    const fetchedData: task = { ...response.taskData };
+    setTaskDetails(fetchedData);
+    setInitialData(fetchedData);
+  }, [isLoading, response]);
+
+  function convertTimeStamp(timeStamp: string) {
+    const dateTime = new Date(parseInt(timeStamp));
+    return dateTime.toLocaleString();
+  }
+
+  function handleChange(
+    event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) {
+    const formData = {
+      [event.target.name]: event.target.value,
+    };
+    setEditedDetials((prv) => {
```

Comment:
```suggestion
    setEditedDetails((prv) => {
```

Reply:
@vinayak-trivedi What's the change you are suggesting here? I don't seem to find any difference. What am I missing?

Reply:
got it, there's a spelling mistake. Thanks for pointing out